joomla/CVE-2023-23752/docker-compose-safe.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  joomla:
+    image: joomla:4.2.8-php8.0
+    restart: always
```

Comment:
```suggestion
```
Let's remove the `restart` so the container doesn't accidentally restart by itself after the test.